### ISSUE TITLE
Incorrect DOF limits for continuous joints

### ISSUE THREAD

**Issue by Stefanos19** [personalrobotics/or_urdf#10]
After loading the urdf files from ada, as specified in https://github.com/personalrobotics/ada/blob/master/adapy/src/adapy/adarobot.py, calling GetDOFLimits results in meaningless values:
```
robot.GetDOFLimits()
Out[1]: 
(array([ 0.   ,  0.   ,  0.   ,  0.785,  0.65 ,  0.   ,  0.   ,  0.   ]),
 array([ 1.05     ,  1.05     ,  0.       ,  5.4981853,  5.6331853,
        0.       ,  0.       ,  0.       ]))
```

**Comment by mkoval** [comment]
There were some major bugs in our handling of continuous joints. I fixed them in #11 and am currently testing this on the URDF model of ADA.

**Comment by mkoval** [comment]
Fixed in #11. I was able to successfully load and plan with the ADA URDF model.
